sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/implementation/models/RestoreOperation.java

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.security.keyvault.administration.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** The RestoreOperation model. */
+@Fluent
+public final class RestoreOperation {
+    /*
+     * Status of the restore operation.
+     */
+    @JsonProperty(value = "status")
+    private String status;
+
+    /*
+     * The status details of restore operation.
+     */
+    @JsonProperty(value = "statusDetails")
+    private String statusDetails;
+
+    /*
+     * Error encountered, if any, during the restore operation.
+     */
+    @JsonProperty(value = "error")
+    private Error error;
+
+    /*
+     * Identifier for the restore operation.
+     */
+    @JsonProperty(value = "jobId")
+    private String jobId;
+
+    /*
+     * The start time of the restore operation
+     */
+    @JsonProperty(value = "startTime")
+    private Long startTime;
+
+    /*
+     * The end time of the restore operation
+     */
+    @JsonProperty(value = "endTime")
+    private Long endTime;
+
+    /**
+     * Get the status property: Status of the restore operation.
+     *
+     * @return the status value.
+     */
+    public String getStatus() {
+        return this.status;
+    }
+
+    /**
+     * Set the status property: Status of the restore operation.
+     *
+     * @param status the status value to set.
+     * @return the RestoreOperation object itself.
+     */
+    public RestoreOperation setStatus(String status) {
```

Comment:
This is a read-only return model. Nothing should be settable.

Reply:
Please refer to my comments above on the matter.